scripts/utils/validateNode.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+################################################################################
+# This program and the accompanying materials are made available under the terms of the
+# Eclipse Public License v2.0 which accompanies this distribution, and is available at
+# https://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Copyright IBM Corporation 2019
+################################################################################
+
+# NODE_HOME Should contain a valid install of Node
+if [[ -n "${NODE_HOME}" ]];
+then
+    ls ${NODE_HOME}/bin | grep node$ > /dev/null
+    if [[ $? -ne 0 ]];
+    then
+        . ${ROOT_DIR}/scripts/utils/error.sh "NODE_HOME: ${NODE_HOME}/bin does not point to a valid install of Node";
+    else
+
+      NODE_OK=`${NODE_HOME}/bin/node -e "console.log('ok')" 2>&1`
+      if [[ ! $NODE_OK == "ok" ]];
+      then 
+        . ${ROOT_DIR}/scripts/utils/error.sh "${NODE_HOME}/bin/node is not functioning correctly";
+      fi
+
+      NODE_MIN_VERSION=6.0
```

Comment:
The minimum version of node is 6.14.4.1 not 6.0

Reply:
bumped it to 6.14, adding check till 4th place